<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="DESKTOP-PSK9T3G" failures="1" tests="1" name="Default test" time="203.417" errors="0" timestamp="2023-09-07T11:43:50 IST">
  <testcase classname="io.cucumber.testng.AbstractTestNGCucumberTests" name="runScenario" time="199.521">
    <failure type="java.lang.NullPointerException" message="Cannot invoke &amp;quot;String.equals(Object)&amp;quot; because &amp;quot;referenceNoFromOmniDataBaseNew&amp;quot; is null">
      <![CDATA[java.lang.NullPointerException: Cannot invoke "String.equals(Object)" because "referenceNoFromOmniDataBaseNew" is null
at stepdefinitions.AdminScreen_Steps_614.User_614ValidatesTheReferenceIdFromTheConfirmationReportAsPerDbOc_Data_Save(AdminScreen_Steps_614.java:808)
at âœ½.user_614 validates the reference id from the confirmation report as per DB oc_data_save(file:///C:/Users/ININDC00614/git/DigitalBanking/Azentio_DigitalBanking_Vimal/src/test/java/features/DBTransactions_614.feature:498)
]]>
    </failure>
  </testcase> <!-- runScenario -->
</testsuite> <!-- Default test -->
